src/govuk/objects/_template.scss

```diff
@@ -12,6 +12,23 @@
     // would like the browser to stay on 100% text zoom by default.
     text-size-adjust: 100%;
 
+    // Add scroll padding to the top of govuk-template unless the exit this page
```

Comment:
I don't follow – if this is to prevent a potential issue with exit this page, why are we only adding the padding when exit this page is _not_ present?

Reply:
I think I got the wording confused here. It should be the other way around, so don't have `scroll-padding` if ETP isn't present. It's a complicated implementation because of how we're trying to cast as wide a net as possible and mitigate as much hassle for folks as possible. I'll update the comment and extend it a bit.